docs/version-history/v23-version.hist

```diff
@@ -12,6 +12,10 @@
   there were ARC CE jobs with no X509userproxy.
   :jira:`2907`
 
+- Fixed a bug that could cause the *condor_shadow* daemon to crash when
+  the job transfer input file list was massive (thousands of files).
```

Comment:
I suspect you can confirm that you only need ~150 files if each filename were ~100 characters long.  Would
```suggestion
  the job transfer input file list was very long (thousands of characters).
```
be better?